lxd/cluster/recover.go

```diff
@@ -507,6 +507,31 @@ func unpackTarball(tarballPath string, destRoot string) error {
 	return nil
 }
 
+func createDatabaseBackup(databaseDir string) (string, error) {
+	varDir := path.Dir(databaseDir)
+
+	// tar interprets `:` as a remote drive; ISO8601 allows a 'basic format'
+	// with the colons omitted (as opposed to time.RFC3339)
+	// https://en.wikipedia.org/wiki/ISO_8601
+	backupFileName := fmt.Sprintf("db_backup.%s.tar.gz", time.Now().Format("2006-01-02T150405Z0700"))
+	tarballPath := filepath.Join(varDir, backupFileName)
+
+	walkDir := path.Base(databaseDir)
+
+	logger.Info("Creating database backup", logger.Ctx{"archive": tarballPath})
```

Comment:
Can we just use "path" here to show where its going to be created. You could change the message to "Creating database backup archive" if the format is important to clarify.